diff --git a/dist/exports/shared/index.js b/dist/exports/shared/index.js
--- a/dist/exports/shared/index.js
+++ b/dist/exports/shared/index.js
@@ -1,6 +1,12 @@
 import * as ce from 'date-fns';
 
+// Patch for missing transpose function
+if (!ce.transpose) {
+  ce.transpose = function transpose(matrix) {
+    return matrix[0].map((_, colIndex) => matrix.map(row => row[colIndex]));
+  };
+}
+
 export function formatDate(date, format = 'MMM d, yyy h:mm bbb') {
   return ce.format(date, format);
 }
